### ISSUE TITLE
Validation Accuracy Does not Change 

### ISSUE THREAD

**Issue by zuh17h** [idiap/attention-sampling#18]
I have used your code, but my problem is that validation accuracy stops after some epochs(at number 65) and does not change. I tried to change all the hyperparameters and it did not work. Have you ever had this problem before? I really appreciate it if you can help.

**Comment by gaowei424** [comment]
same case
